src/main/java/org/takes/facets/auth/PsBasic.java

```diff
@@ -51,6 +52,12 @@
 public final class PsBasic implements Pass {
 
     /**
+     * URN format.
+     * @checkstyle ConstantUsageCheck (2 lines)
```

Comment:
@yegor256  I don't know why checkstyle says I'm not using this constant. I have to add this ignore in `@checkstyle`